### ISSUE TITLE
m3u8文件播放和seek问题

### ISSUE THREAD

**Issue by dianyou1215** [Bilibili/ijkplayer#1314]
优酷的fkv, ts m3u8文件播放中间卡住不能再次播放，所有的flv，ts的m3u8文件都不能seek。

**Comment by dianyou1215** [comment]
@bbcallen 能告诉下我怎么改吗？谢谢！

**Comment by Android4MediaPlayer** [comment]
@dianyou1215 请提供下播放地址或更详细的信息

**Comment by dianyou1215** [comment]
视频地址：http://182.92.4.93:1122/VideoParse/YoukuM3U8/XNjAzOTI4OTU2/flvhd，播放过程会卡住，不卡住的时候没法seek，或者seek后卡住不播放。bbc好像说要改个底层文件，我没搞过，指点下，谢谢！

**Comment by dianyou1215** [comment]
@Android4MediaPlayer @bbcallen 
视频地址：http://182.92.4.93:1122/VideoParse/YoukuM3U8/XNjAzOTI4OTU2/flvhd，播放过程会卡住，不卡住的时候没法seek，或者seek后卡住不播放。bbc好像说要改个底层文件，我没搞过，指点下，谢谢！

**Comment by Android4MediaPlayer** [comment]
@dianyou1215 需要怎样操作呢 ？我测试是没有问题的，不知道怎么帮你

**Comment by bbcallen** [comment]
Seek works well in demo project with your url.

**Comment by dianyou1215** [comment]
我用的是0.4.5版本的ijk， build是用的是module-lite.sh，怎么不行呢，奇怪！

**Comment by dianyou1215** [comment]
@bbcallen 我用的是0.4.5版本的ijk， build是用的是module-lite.sh，怎么不行呢，奇怪！之前看别人遇到类似的问题，你的回答是在底层修改 libavformat/hls.c的代码。我做过不太明白，请指教下，谢谢！

**Comment by dianyou1215** [comment]
@bbcallen  是android版本的！

**Comment by bbcallen** [comment]
Have you tried 0.5.1?

**Comment by dianyou1215** [comment]
我们有是最新的版本，在群里有人说新版有崩溃老版本稳定点，你用的 build是用的是module-lite.sh吗？我用0.4.5build是用的是module-lite.sh，有这个问题

**Comment by nanguaSky** [comment]
那个问题我还是没能解决，我也是build 最新的0.5.1版本，后来干脆用了个折中的办法，卡主10秒直接释放资源从新setVideoUrl  然后seekTo  卡住的地方

**Comment by UC10D** [comment]
@nanguaSky @dianyou1215 哥哥们，解决了吗？我也出现这个问题了

**Comment by Android4MediaPlayer** [comment]
give out playing url？

**Comment by UC10D** [comment]
http://121.46.19.92/ipad?file=/126/161/4lEFV0BgT02QK92xPlfFME.mp4&start=135.4&end=152.066&sig=I-I1PCrfPiFaFeTjGptRbISaj6dAbjGM&ch=my&cateCode=127119&vid=84105470&uid=null&plat=15&pt=6&prod=ott&pg=1&eye=0&cv=&qd=816

**Comment by Android4MediaPlayer** [comment]
key fame problem

**Comment by UC10D** [comment]
how to deal with

**Comment by UC10D** [comment]
when i seekto a long distance,over it's buffering,mediaplayer just stop.

**Comment by Android4MediaPlayer** [comment]
That was a bug.

**Comment by Android4MediaPlayer** [comment]
Have you tried 0.5.1?

**Comment by UC10D** [comment]
yes, it's 0.5.1. i tride the demo,the same problem.And i tride the system mediaplayer, it's ok.

**Comment by Android4MediaPlayer** [comment]
I think It was stopped for the system mediaplayer.

**Comment by Android4MediaPlayer** [comment]
That was not ok.

**Comment by UC10D** [comment]
no, i tried and it‘s ok.Some phones have support m3u8.

**Comment by Android4MediaPlayer** [comment]
It is ts that "http://121.46.19.92/ipad?file=/126/161/4lEFV0BgT02QK92xPlfFME.mp4&start=135.4&end=152.066&sig=I-I1PCrfPiFaFeTjGptRbISaj6dAbjGM&ch=my&cateCode=127119&vid=84105470&uid=null&plat=15&pt=6&prod=ott&pg=1&eye=0&cv=&qd=816"

**Comment by UC10D** [comment]
thank you

**Comment by UC10D** [comment]
this bug will be solve?
